resources/js/app-layout.js

```diff
@@ -83,6 +90,18 @@ window.ProcessMaker.alert = function (msg, variant, showValue = 60) {
     })
 };
 
+// Setup our login modal
+window.ProcessMaker.sessionModal = function (title, message, time) {
+    ProcessMaker.navbar.sessionTitle = title || __("Session Warning");
+    ProcessMaker.navbar.sessionMessage = message || __("Your session is about to expire.");
```

Comment:
Is this message being used?

Reply:
No, but it is a default message for the modal